.moai/specs/SPEC-EVAL-LOOP-001/plan.md

```
@@ -0,0 +1,177 @@
+    evaluator: true
+    sprint_contract: false
+    # NEW: feedback-loop config
+    evaluator_mode: feedback-loop  # alternatives: final-pass, feedback-loop
+    max_iterations: 2
+    improvement_threshold: 0.10
+    stagnation_consecutive: 1
+```
+
+### 4.2 Protocol Pseudocode (evaluator-active body 문서화 예정)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)